gnomad_qc/v4/create_release/create_de_novo_release.py

```diff
@@ -42,304 +31,210 @@
+        # release sites and some of the probands weren't included in the release.
+        worst_csq_for_variant=ht.vep.worst_csq_for_variant.most_severe_consequence,
+        gene_id=ht.vep.worst_csq_for_variant.gene_id,
+        transcript_id=ht.vep.worst_csq_for_variant.transcript_id,
+        gnomAD_v4_exomes_AF=freq_ht[ht.key].freq[0].AF,
+    ).drop("vep")
+
+    # Store the de novo calls with all confidence levels.
+    # TODO: Should this be after non-star filtering or even after pass_filters?
+    ht_all_conf = ht
+
+    ht = ht.filter(~ht.alt_is_star).checkpoint(new_temp_file("denovo_no_star", "ht"))
+    logger.info(f"Getting {ht.count()} de novos after filtering out '*' alts...")
+
+    ht = ht.filter(ht.pass_filters).checkpoint(
+        new_temp_file("denovo_pass_filters", "ht")
+    )
+    logger.info(
+        f"Getting {ht.count()} de novos after AS_VQSR and InbreedingCoeff "
+        f"filters..."
+    )
 
-    :return: Config dict.
-    """
-    config = get_config(data_type="exomes")
-    config["info"].update(
-        {
-            "select": ["was_split", "a_index"],
-            "custom_select": custom_info_select,
-        }
+    ht = ht.filter(ht.de_novo_AC.AC_high_conf > 0).checkpoint(
+        new_temp_file("denovo_high_conf", "ht")
     )
-    config.update(
-        {
-            "filters": {
-                "ht": final_filter(all_variants=True, only_filters=True).ht(),
-                "path": final_filter(all_variants=True, only_filters=True).path,
-                "select": ["filters"],
-            },
-            # TODO: Change when we decide on the final de novo data to release.
-            "de_novos": {
-                "ht": get_trio_stats(releasable_only=True).ht(),
-                "path": get_trio_stats(releasable_only=True).path,
-                "custom_select": custom_de_novo_select,
-            },
-            "joint_freq": {
-                "ht": public_release("joint").ht(),
-                "path": public_release("joint").path,
-                "custom_select": custom_joint_freq_select,
-                "custom_globals_select": custom_joint_freq_select_globals,
-            },
-            "exomes_an": {
-                "ht": all_sites_an("exomes").ht(),
-                "path": all_sites_an("exomes").path,
-                "custom_select": custom_an_select,
-                "custom_globals_select": custom_an_select_globals,
-            },
+    logger.info(f"Getting {ht.count()} high confidence de novos...")
+
+    # Get the set of coding variant except for the splice region variants for
+    # calibration.
+    coding_csqs = hl.literal(
+        set(CSQ_CODING_HIGH_IMPACT + CSQ_CODING_MEDIUM_IMPACT + CSQ_CODING_LOW_IMPACT)
+        - {
+            "splice_polypyrimidine_tract_variant",
+            "splice_donor_5th_base_variant",
+            "splice_region_variant",
+            "splice_donor_region_variant",
```

Comment:
maybe you should also remove `incomplete_terminal_codon_variant` ("A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed"), `coding_sequence_variant` ("A sequence variant that changes the coding sequence"), and `coding_transcript_variant` ("A transcript variant of a protein coding gene") here

Reply:
for consistency then, let's keep `coding_sequence_variant`